### PR TITLE
Refactor vars and add prometheus-exporter alarms

### PR DESCRIPTION
**Pull request by Wasaac**
Move alarm definitions to vars file. Extend alarms with prometheus exporter metric based alarm set.

### CODE REVIEW

**Review by dougszumski — COMMENTED** [review]
Hmm, is there a good reason for the refactoring? It's a bit of a pain to review the *actual* changes.

**Comment by Wasaac** [comment]
> Hmm, is there a good reason for the refactoring? It's a bit of a pain to review the _actual_ changes.

Fair point, I should have split that into another PR. The refactor removes code duplication, reduces the total size of the repo & improves alarm definition readability. I'd argue it's some extra effort now to save more later.

File: tasks/main.yml [hunk 1]
```diff
@@ -30,33 +31,69 @@
...
-    - system_alarms
-    - monasca_alarms
-    - openstack_alarms
-    - service_alarms
+    - always
   register: default_notification
 
 # Include the various alarm sets
-- import_tasks: system.yml
+- include_tasks: alarms.yml
```
Comment (dougszumski) [inline tasks/main.yml:38]: I suppose you could loop over the different groups here, but no need to over-complicate it.

**Review by dougszumski — APPROVED** [review]
Thanks Isaac. I agree with your comments about the refactor.

File: tasks/main.yml (hunk 1 above)
Comment (Wasaac) [reply tasks/main.yml:38]: That'd be nicer certainly. I couldn't figure out how to preserve the original tags inside a loop though.

**Review by bbezak — APPROVED** [review]

**Review by oneswig — APPROVED** [review]
Looks good, thanks @Wasaac

**Pull request merged**